### PR TITLE
chore(babel-preset-gatsby): Adds the missing peer dependency on @babel/core

### PR DESCRIPTION
**Pull request by arcanis**
## Description

The Babel plugins have a peer dependency on `@babel/core`, and since `babel-preset-gatsby` doesn't provide it it needs to list it as a peer dependency itself.

### CODE REVIEW

File: packages/babel-preset-gatsby/package.json [hunk 1]
```diff
@@ -10,6 +10,9 @@
     "@babel/preset-react": "^7.0.0",
     "babel-plugin-macros": "^2.4.2"
   },
+  "peerDependencies": {
+    "@babel/core": "*"
```
Comment (DSchau) [inline packages/babel-preset-gatsby/package.json]: Should we be relatively safer here and pin to `^7.0.0`?

Comment (arcanis) [reply packages/babel-preset-gatsby/package.json]: Sure! Will update the PR - since the plugins are scoped to `^7.0.0` it should be fine 👍

**Comment by arcanis** [comment]
The fail looks unrelated

**Comment by DSchau** [comment]
@arcanis it is. Looking at it now.

**Comment by sidharthachatterjee** [comment]
Published in babel-preset-gatsby@0.1.8

**Pull request merged**